examples/src/main/kotlin/examples/dataset/PreprocessOnlyOneImage.kt

```diff
@@ -23,48 +26,44 @@ import javax.swing.JFrame
 fun main() {
     val preprocessedImagesDirectory = File("processedImages")
 
-    val preprocessing: Preprocessing = preprocess {
-        transformImage {
-            crop {
-                left = 100
-                right = 0
-                top = 100
-                bottom = 0
-            }
-            rotate {
-                degrees = 0f
-            }
-            resize {
-                outputWidth = 400
-                outputHeight = 400
-                interpolation = InterpolationType.NEAREST
-                save {
-                    dirLocation = preprocessedImagesDirectory
-                }
-            }
-            pad {
-                top = 10
-                bottom = 40
-                left = 10
-                right = 10
-                mode = PaddingMode.Fill(Color.WHITE)
-            }
-            convert { colorMode = ColorMode.BGR }
+    val preprocessing = pipeline<BufferedImage>()
+        .crop {
+            left = 100
+            right = 0
+            top = 100
+            bottom = 0
         }
-        transformTensor {
-            rescale {
-                scalingCoefficient = 255f
+        .rotate {
+            degrees = 0f
+        }
+        .resize {
+            outputWidth = 400
+            outputHeight = 400
+            interpolation = InterpolationType.NEAREST
+            save {
+                dirLocation = preprocessedImagesDirectory
             }
         }
-    }
+        .pad {
+            top = 10
+            bottom = 40
+            left = 10
+            right = 10
+            mode = PaddingMode.Fill(Color.WHITE)
+        }
+        .convert { colorMode = ColorMode.BGR }
+        .toFloatArray { }
```

Comment:
I think it's useful to show users how to manually visualize the result of preprocessing

Reply:
We already have `FloatArray` visualization in the [other example](https://github.com/ermolenkodev/KotlinDL/blob/416/examples/src/main/kotlin/examples/dataset/ImagePreprocessing.kt) (and there it's more justified since it's a sample from the dataset).